pkg/util/indexparamcheck/index_type.go

```diff
@@ -52,6 +52,12 @@ const (
 	AutoIndex IndexType = "AUTOINDEX"
 )
 
+func IsScalarIndexType(indexType IndexType) bool {
+	return indexType == IndexSTLSORT || indexType == IndexTRIE || indexType == IndexTrie ||
+		indexType == IndexBitmap || indexType == IndexHybrid || indexType == IndexINVERTED ||
+		indexType == AutoIndex
```

Comment:
AutoIndex can also be VectorIndex.

Reply:
OK, remove it, actually userindexparam not include autoindex